sickchill.json

```
@@ -0,0 +1,54 @@
+{
+	"SickChill": {
+		"containers": {
+			"SickChill": {
+				"image": "linuxserver/sickchill",
+				"launch_order": 1,
+				"ports": {
+					"8081": {
+						"description": "Port for SickChill Web interface. Suggested default: 8081.",
+						"host_default": 8081,
+						"label": "WebUI port",
+						"protocol": "tcp",
+						"ui": true
+					}
+				},
+				"volumes": {
+					"/download": {
+						"description": "Choose a share to download to (also known as the sickchill monitor folder). It is important that the owner/group specified later have full access to the share (R/W/E).",
+						"label": "Download"
+					},
+					"/config": {
+						"description": "Choose a share to host the configuration settings for SickChill. It is important that the owner/group specified later have full access to the share (R/W/E).",
+						"label": "Config"
+					},
+					"/tv": {
+						"description": "Choose a share to host shows (also known as parent folder). You can add additional shares later via the RockOn's settings menu. It is important that the owner/group specified later have full access to the share (R/W/E).",
+						"label": "TV"
+					}
+				},
+				"environment": {
+					"PUID": {
+						"description": "Enter a valid UID to run SickChill under. It must have full permissions to all Shares mapped in the previous step.",
+						"label": "UID",
+						"index": 1
+					},
+					"PGID": {
+						"description": "Enter a valid GID to use along with the above UID. It(or the above UID) must have full permissions to all Shares mapped in the previous step.",
+						"label": "GID",
+						"index": 2
+					}
+				}
+			}
+		},
+		"description": "Automatic Video Library Manager for TV Shows",
+		"more_info": "Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.<p>Uses custom docker image: <a href='https://hub.docker.com/r/linuxserver/sickchill/' target='_blank'>https://hub.docker.com/r/linuxserver/sickchill/</a>, available for amd64, arm32v7 and arm64v8.",
+		"ui": {
+			"slug": ""
+		},
+		"volume_add_support": true,
+		"website": "https://github.com/SickChill/SickChill",
```

Link to project's website rather than Github page, which we can reserve for the "more_info" field:
```suggestion
		"website": "https://sickchill.github.io/",
```